oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/util/LoggingDocumentStoreWrapper.java

```diff
@@ -283,6 +283,7 @@ public T call() throws Exception {
     }
 
     @Override
+    @NotNull
```

Comment:
nitpicking: IIUC this is unrelated to this branch (and mostly code beautification). I would usually tend to avoid overloading a PR or a branch with unrelated aspects, as it might later complicate backporting. Maybe not with a one liner like here, hence nitpicking :)